docs/settings/table-layout.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+---
+
+## Table layout
+
+By default, tables in vanilla use table-layout: fixed. It is faster, and ensures more predictable and controllable table behaviour, but requires setting widths explicitly on the first row of the table.
```

Comment:
- Please capitalise Vanilla.
- Please add back ticks around "table-layout: fixed"

Reply:
I would prefer if the default table behavior was auto and if the user wants more control and is willing to add widths to the table columns then they switch `fixed`.

Reply:
I do feel the `$table-layout-fixed` variable is unnecessary and overengineered. Defaulting to `auto` with a utility class to override should be quite sufficient and is more declarative and less surprising.

Reply:
The default has been 'fixed' for a while. Changing this may lead to regressions, especially in maas.

Reply:
@anthonydillon @nottrobin table-layout has been set to fix since 1.7.1 and ensures tables behave predictably. This can potentially cause lots of regressions. Are you sure you want to change this?

This pr was simply meant to give flexibility to use auto for automatically generated tables from discourse on docs, in response to this [issue](https://github.com/canonical-web-and-design/vanilla-framework/blame/22d753c8a0cbce3481dc00bfd6c82ed84785a553/scss/_base_tables.scss).

I just realised that the doc wording makes it sound a lot more ominous than it actually is. When I wrote ```requires setting widths explicitly``` I meant it requires it if you're not happy with the default browser behaviour in this case, which is:

> Table and column widths are set by the widths of table and col elements or by the width of the first row of cells. Cells in subsequent rows do not affect column widths. [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout)

I've taken that part out.